components/Matches/Matches.js

```diff
@@ -27,38 +28,30 @@ export default class MatchesScreen extends React.Component {
         headerRightContainerStyle: styles.rightNavIcon,
       })
 
-    render() {
-        return (
-              <ScrollView>
-                <View >
-                  <View style={styles.matches}>
-                    <Match matches={sampleMatches} />
+      
+      render() {
+        if (this.props.matches.length) {
```

Comment:
Great job on making the matches page dynamic and having a default message when there are no matches yet